### ISSUE TITLE
Metrics/AbcSize measuring something else?

### ISSUE THREAD

**Issue by tmornini** [bbatsov/rubocop#1761]
Hey there. 0.29.1 here.

This code:

```
def to_s
  [
    format('%7d', first_block_in_era),
    format('%10d', reward_era_number),
    align(btc_per_block, 2),
    align(year, 4, 3),
    align(start_btc),
    align(btc_added),
    align(end_btc),
    percentage(btc_increase_percentage, 4, 8),
    percentage(end_btc_percent_of_limit, 8, 8),
    percentage(supply_inflation_rate, 11, 8)
  ].join ' : '
end
```

is generating this warning:

```
lib/reward_era.rb:88:3: C: Metrics/AbcSize: Assignment Branch Condition size for to_s is too high. [21/15]
  def to_s
  ^^^
```

This doesn't make a lot of sense to me. There are no assignments and no branch
conditions in this code!

**Comment by jonas054** [comment]
I can see how you would interpret "branch" to mean alternative routes through the method, but that's not what it means in the [ABC metric](http://c2.com/cgi/wiki?AbcMetric). As you see on that wiki page, it's method calls.

**Comment by tmornini** [comment]
Hmmm. Leaves me wondering how such summary operations are to be performed. :-)

Thanks!
